ui/components/multichain/token-list-item/token-list-item.tsx

```diff
@@ -285,7 +283,8 @@ export const TokenListItem = ({
               size={AvatarNetworkSize.Xs}
               name={allNetworks?.[chainId as Hex]?.name}
               src={tokenChainImage || undefined}
-              backgroundColor={testNetworkBackgroundColor}
+              backgroundColor={BackgroundColor.backgroundDefault}
+              borderWidth={2}
```

Comment:
Before: 
<img width="62" alt="Screenshot 2024-12-18 at 13 40 01" src="https://github.com/user-attachments/assets/29510940-69e3-49d8-8bfc-4a6ffdf0905b" />

After: 
<img width="60" alt="Screenshot 2024-12-18 at 13 11 18" src="https://github.com/user-attachments/assets/6b52555d-1de7-4c91-9e44-7c22f4ebd273" />

CSS breakdown: 
<img width="1904" alt="Screenshot 2024-12-18 at 13 20 28" src="https://github.com/user-attachments/assets/382786af-91c2-478f-8462-580f17eaeb2f" />

Reply:
Hey @DDDDDanica, should the before and after images be swapped? For example, should the 2px border be *after* and the 1px border be *before*?

Reply:
Good spot ! switched 🙏🏻